### PR TITLE
win32: explicitly guard dark mode calls by Windows version

### PR DESCRIPTION
**Pull request by kasper93**
In theory it shouldn't be needed, but apparently on Windows 7 those ordinals returns pointers to not executable code trigerring DEP.

Fixes: #11610
Fixes: 9feeb324eddd9ed73ae667e10275f663d70f7544

### CODE REVIEW

**Review by jeeb — CHANGES_REQUESTED** [review]
Might make sense to look into limiting dark mode a bit more than just Win10 generically looking at uxtheme nr135 being different with earlier versions of Win10 :) .

File: video/out/w32_common.c [hunk 1]
```diff
@@ -1543,7 +1544,9 @@ static void w32_api_load(struct vo_w32_state *w32)
                 (void *)GetProcAddress(imm32_dll, "ImmDisableIME");
 
     // Dark mode related functions, avaliable since a Win10 update
-    HMODULE uxtheme_dll = GetModuleHandle(L"uxtheme.dll");
+    // Explicitly check the version to workaround Windows 7 crashes
+    HMODULE uxtheme_dll = !IsWindows10OrGreater() ? NULL :
```
[the remark this answers is not in the document]
Comment (kasper93) [reply video/out/w32_common.c]: done

**Review by Jj0YzL5nvJ — APPROVED** [review]
This looks ready, what's holding it up?

**Comment by Dudemanguy** [comment]
>This looks ready, what's holding it up?

Nobody knowing anything about windows. Looks OK I guess?

**Comment by kasper93** [comment]
I think @jeeb wanted to not use `RtlGetNtVersionNumbers`, but I don't have nicer way to check the version. Doing file version is a lot of unnecessary boilerplate code.

**Pull request merged**